### ISSUE TITLE
User Request - When messaging or inviting a friend, make keyboard automatically appear

### ISSUE THREAD

**Issue by nylim11** [ceaseless-prayer/CeaselessIOS#185]
A user did not know that they could edit the text of the message before sending. In order to make it clear to the user that the text is able to be personalized and changed,  can we have the keyboard automatically appear as well when tapping on the paper airplane icon or inviting a person in Ceaseless?

**Comment by calvinf** [comment]
I don't think you want it to appear unless they're editing the field. For that you need to make clear in the UX what fields are editable. Although if you autofocus the field it would make sense to open keyboard right away.

**Comment by nylim11** [comment]
This request is related to https://github.com/ceaseless-prayer/CeaselessIOS/issues/169
